colorspage/colorspage.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Colors |Futbolt UI</title>
+    <link rel="stylesheet" href="/documentation/documentation.css" />
+    <script src="https://kit.fontawesome.com/0a41da8839.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="/components/component.css" />
+    <link rel="stylesheet" href="/colorspage/colorspage.css">
+</head>
+
+<body>
+    <!-- || Container  -->
+    <div class="container">
+        <!-- || Navbar starts -->
+        <nav class="navbar flx-row flx-main-center">
+            <div class="navbar-logo">
+                <img src="/assets/images/futbolt-lg.png" alt="" loading="eager" />
+            </div>
+            <div class="navbar-title">Futbolt UI</div>
+            <div class="navbar-links">
+                <ul>
+                    <li><a href="/index.html">Home</a></li>
+                    <li><a href="#" class="navbar-links-active">Docs</a></li>
+                    <li>
+                        <a href="https://github.com/sks-21/Futbolt-Component-Library">
+                            <i class="fab fa-github"></i> Github</a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+
+        <!-- || Side Menu starts   -->
+        <aside class="menu flx-col flx-main-center">
+            <ul class="menu-list">
+                <li class="menu-list-title">Getting Started</li>
+                <li class="menu-list-item-active">
+                    <a href="#">Installation</a>
+                </li>
+                <li>
+                    <a href="/typography/typography.html">Typography</a>
```

Comment:
can you use ./ at start please try this better practice according to me

Reply:
Ok